pkg/motor/create_account.go

```diff
@@ -9,29 +9,26 @@ import (
 	"github.com/sonr-io/sonr/pkg/did"
 	"github.com/sonr-io/sonr/pkg/tx"
 	"github.com/sonr-io/sonr/pkg/vault"
-	ct "github.com/sonr-io/sonr/third_party/types/common"
 	mt "github.com/sonr-io/sonr/third_party/types/motor/api/v1"
 	rt "github.com/sonr-io/sonr/x/registry/types"
 )
 
 func (mtr *motorNodeImpl) CreateAccount(request mt.CreateAccountRequest) (mt.CreateAccountResponse, error) {
 	// create motor
-	mtr.callback.OnMotorEvent(ct.MotorCallbackMessage_MTR_INIT, false)
+	mtr.callback.OnMotorEvent("Initializing motor", false)
```

Comment:
References to the value represented by each string should be made through some kind of strongly typed value / constant